modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -50,24 +67,60 @@ spec:
   resolution: DNS
 ----
 +
-You can use the `oc apply` command with either of the following methods to apply the objects:
+.`DestinationRule` CR
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: destination-rule-threescale-saas-system
+spec:
+  host: multitenant.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: multitenant.3scale.net
+----
++
+Use the following commands to apply external `ServiceEntry` and related `DestinationRule` CRs to your cluster:
++
+.3scale Hosted backend
+** Save the *backend* objects to one or more files, and then use the following syntax:
++
+[source,terminal]
+----
+$ oc apply -f <service-entry-threescale-saas-backend.yml>
+$ oc apply -f <destination-rule-threescale-saas-backend.yml>
+----
 
-** Save the objects to one or more files, and then use the following syntax:
+** To apply the *backend* objects without first saving them to a file, use the following command:
++
+[source,terminal]
+----
+$ echo -n "<service-entry-threescale-saas-backend.yml>" | oc apply -f -
+$ echo -n "<destination-rule-threescale-saas-backend.yml>" | oc apply -f -
+----
+
++
+.3scale Hosted system
+** Save the *system* objects to one or more files, and then use the following syntax:
 +
 [source,terminal]
 ----
-$ oc apply -f <filename.yml>
+$ oc apply -f <service-entry-threescale-saas-system.yml>
+$ oc apply -f <destination-rule-threescale-saas-system.yml>
 ----
 
-** To apply the objects without first saving them to a file, use the following command:
+** To apply the *system* objects without first saving them to a file, use the following command:
 +
 [source,terminal]
 ----
-$ echo -n "<filename.yml>" | oc apply -f -
+$ echo -n "<service-entry-threescale-saas-system.yml>" | oc apply -f -
+$ echo -n "<destination-rule-threescale-saas-system.yml>" | oc apply -f -
 ----
 
-Alternatively, you can deploy an in-mesh 3scale service. To do this, change the location of these services in the custom resources.
+Alternatively, you can deploy an in-mesh 3scale service. To deploy an in-mesh 3scale service, change the location of the services in the CR by deploying 3scale and linking to the deployment.
```

Comment:
@rahulanand16nov  Is this alternative way to deploy an in-mesh 3scale service necessary to document here or would it be better to remove this line so as not to cause confusion?